app/index.js

```diff
@@ -726,9 +776,68 @@ module.exports = generators.Base.extend({
     }
   },
 
+  generateSDKs: function() {
```

Comment:
Ok cool, I think most of the SDK generator work can go before the `writing` phase. I believe the Package.swift additions are already in the right spot within the `writing` phase.